sycl/unittests/helpers/PiMockPlugin.hpp

```diff
@@ -432,6 +432,145 @@ inline pi_result mock_piMemImageCreate(pi_context context, pi_mem_flags flags,
   return PI_SUCCESS;
 }
 
+inline pi_result
```

Comment:
will a test be added, in test-e2e ?

Reply:
Yes, as our final [4/4] PR which will depend on this and the next.